Ball of Duty Server/Domain/Maps/Map.cs

```diff
@@ -174,10 +176,34 @@ public GameObjectDTO[] ExportGameObjects()
                 select new GameObjectDTO
                 {
                     Id = go.Id,
-                    Body = body
+                    Body = body,
+                    Specialization = GetSpecialization(go)
                 }).ToArray();
         }
 
+        /// <summary>
+        /// Returns 0 if the specialization
+        /// </summary>
+        /// <param name="go"></param>
+        /// <returns></returns>
+        private int GetSpecialization(GameObject go)
```

Comment:
pls move out of Map.cs, to another file in specializations or make a method on character returning this int value.

ex. `switch (go.GetType())`
`case typeof(Blaster):`
`default: `